src/main/webapp/app/exercises/programming/participate/programming-submission.service.ts

```diff
@@ -13,6 +13,8 @@ import { getLatestSubmissionResult, setLatestSubmissionResult, SubmissionType }
 import { ProgrammingExerciseStudentParticipation } from 'app/entities/participation/programming-exercise-student-participation.model';
 import { findLatestResult } from 'app/shared/util/utils';
 import { Participation } from 'app/entities/participation/participation.model';
+import { FileUploadSubmission } from 'app/entities/file-upload-submission.model';
```

Comment:
Some small GitHub Actions suggestions, not used imports. Please remove them

Reply:
Thanks for noticing that!